Admin/Test-ExchangePropertyPermissions/Test-ExchangePropertyPermissions.ps1

```diff
@@ -0,0 +1,232 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
```

Comment:
Do we want a description help section here? Or just have the docs cover it?

Reply:
I think the doc is fine for now.